maintainers/maintainer-list.nix

```diff
@@ -14112,4 +14112,10 @@
     github = "nigelgbanks";
     githubId = 487373;
   };
+  brianmcgee = {
```

Comment:
This should be a separate commit: `maintainers: add brianmcgee`

https://github.com/NixOS/nixpkgs/blob/a15fbdb88f38800c8e554772a12de8031815802a/maintainers/maintainer-list.nix#L49

Please sort the new entry alphabetically.